### PR TITLE
chore: rename change to modify

### PR DESCRIPTION
**Pull request by CookiePieWw**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## Refer to a related PR or issue link (optional)

proto: https://github.com/GreptimeTeam/greptime-proto/pull/200

## What's changed and what's your intention?

Just a rename.

Rename `ChangeColumnType` to `ModifyColumnType` to match it with SQL syntax.

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

### CODE REVIEW

**Review by MichaelScofield — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -122,7 +122,7 @@ etcd-client = "0.13"
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "e1070ad3e7ad3d89f8dd1e20756b0c0cc26f3365" }
+greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "be5f5f21e0b4eafe93de6db3ae870c1c982fef9f" }
```
Comment (WenyXu) [inline Cargo.toml:125]:
```suggestion
greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "fb4e2146b1ea8816148304f1f258e7a73736a905" }
```

File: Cargo.lock [hunk 2]
```diff
@@ -4595,7 +4595,7 @@ dependencies = [
 [[package]]
 name = "greptime-proto"
 version = "0.1.0"
-source = "git+https://github.com/GreptimeTeam/greptime-proto.git?rev=e1070ad3e7ad3d89f8dd1e20756b0c0cc26f3365#e1070ad3e7ad3d89f8dd1e20756b0c0cc26f3365"
+source = "git+https://github.com/GreptimeTeam/greptime-proto.git?rev=be5f5f21e0b4eafe93de6db3ae870c1c982fef9f#be5f5f21e0b4eafe93de6db3ae870c1c982fef9f"
```
Comment (WenyXu) [inline Cargo.lock:4598]:
```suggestion
source = "git+https://github.com/GreptimeTeam/greptime-proto.git?rev= fb4e2146b1ea8816148304f1f258e7a73736a905#fb4e2146b1ea8816148304f1f258e7a73736a905"
```

**Comment by CookiePieWw** [comment]
Captured db#5000 and proto#200 flag! Quite lucky for me :)

**Comment by WenyXu** [comment]
> Captured db#5000 and proto#200 flag! Quite lucky for me :)

Interesting😜

**Pull request merged**